spec/integration/make/dependencies_spec.rb

```diff
@@ -11,6 +11,14 @@
       expect(app_dependencies_in_compose_file(project_name))
         .to match_array(app_dependencies_in_makefile(project_name))
     end
+
+    it "has only valid dependencies in the #{project_name} Makefile" do
+      app_dependencies = MakefileHelper.dependencies(project_name)
+        .reject { |dep| dep =~ /^(bundle|clone)/ }
+        .map { |dep| "#{dep}-app" }
```

Comment:
Optional: you could avoid this by instead comparing with `ProjectsHelper.all_projects`.